### PR TITLE
Change Cucumber profiles by Cucumber tags, to support the union of functional areas

### PR DESCRIPTION
**Pull request by srbarrios**
Related to https://github.com/uyuni-project/uyuni/pull/3597

### CODE REVIEW

**Review by lkotek — APPROVED** [review]
:+1:

File: jenkins_pipelines/environments/common/pipeline.groovy [hunk 1]
```diff
@@ -71,8 +71,8 @@ def run(params) {
...
             }
             stage('Secondary features') {
-                def statusCode1 = sh script:"./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'export LONG_TESTS=${params.long_tests}; export PROFILE=${params.functional_scope}; cd /root/spacewalk/testsuite; rake cucumber:secondary'", returnStatus:true
-                def statusCode2 = sh script:"./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'export LONG_TESTS=${params.long_tests}; export PROFILE=${params.functional_scope}; cd /root/spacewalk/testsuite; rake ${params.rake_namespace}:secondary_parallelizable'", returnStatus:true
+                def statusCode1 = sh script:"./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'export LONG_TESTS=${params.long_tests}; export TAGS=$ {params.functional_scopes}; cd /root/spacewalk/testsuite; rake cucumber:secondary'", returnStatus:true
```
Comment (ktsamis) [inline jenkins_pipelines/environments/common/pipeline.groovy:74]: the space between $ and the { looks wrong

File: jenkins_pipelines/environments/manager-4.0-dev-acceptance-tests-NUE [hunk 2]
```diff
@@ -20,7 +20,7 @@ node('sumaform-cucumber') {
...
+            extendedChoice(name: 'functional_scopes', defaultValue: 'default', multiSelectDelimiter: ',', quoteValue: false, saveJSONParameterToFile: false, type: 'PT_CHECKBOX', visibleItemCount: 30, value: 'default,scope_smdba,scope_spacecmd,scope_spacewalk_utils,scope_visualization,scope_notification_message,scope_virtual_host_manager,scope_subscription_matching,scope_formulas,scope_sp_migration,scope_cve_audit,scope_onboarding,scope_content_lifecycle_management,scope_res,scope_recurring_actions,scope_maintenance_windows,scope_cluster_management,scope_building_container_images,scope_kubernetes_integration,scope_openscap,scope_ubuntu,scope_action_chains,scope_salt_ssh,scope_tomcat,scope_changing_software_channels,scope_monitoring,scope_salt,scope_cobbler,scope_sumatoolbox,scope_virtualization,scope_hub,scope_retail,scope_configuration_channels,scope_content_staging,scope_proxy,scope_traditional_client,scope_xmlrpc,scope_power_management,scope_retracted_patches', description: 'Choose the functional scopes that you want to test')
```
Comment (ktsamis) [inline jenkins_pipelines/environments/manager-4.0-dev-acceptance-tests-NUE:23]: nitpick, spaces after the commas please?

**Review by ktsamis — CHANGES_REQUESTED** [review]

File: jenkins_pipelines/environments/manager-4.0-dev-acceptance-tests-NUE (hunk 2 above)
Comment (srbarrios) [reply jenkins_pipelines/environments/manager-4.0-dev-acceptance-tests-NUE:23]: No, we can't this is implementation design of this plugin.

**Pull request merged**